pkg/pdp/service/get_proofset.go

```diff
@@ -48,7 +48,8 @@ func (p *PDPService) ProofSet(ctx context.Context, id int64) (*ProofSet, error)
 
 	// Step 5: Build the response.
 	response := &ProofSet{
-		ID:                 proofSet.ID,
+		ID: proofSet.ID,
+		// TODO this will panic if ProveAtEpoch is nill, which it is when the proofset is first created
```

Comment:
Not caused by this change, but found during testing - https://github.com/storacha/piri/issues/111